static_frame/core/store.py

```diff
@@ -152,6 +185,45 @@ def label_decode(self, label: str) -> tp.Hashable:
         ]
 
 
+class StoreConfigHE(StoreConfig):
+    '''
+    A hash/equals subclass of :obj:`StoreConfig`, permiting usage in a Python set, dictionary, or other contexts where a hashable container is needed.
+
+    To support hashability, label_encoder/label_decoder are ignored
+    '''
+    __slots__ = ('_hash',)
```

Comment:
Maybe it is easier of `StoreConfigHE` is not a subclass of StoreConfig, as it has a subset of its attrs. Actually, now that I think about it `StoreConfig` should be a subclass of `StoreConfigHE`, as it expands the attrs available. We can then override `__hash__` in `StoreConfig` to raise.

Reply:
I wondered about that, but I wasn't sure. That will definitely clean things up.